tests/fixtures/on_demand_cluster_fixtures.py

```diff
@@ -79,7 +79,11 @@ def ondemand_https_cluster_with_auth():
 
 @pytest.fixture(scope="session")
 def multinode_cpu_cluster():
-    args = {"name": "rh-cpu-multinode", "num_instances": 2, "instance_type": "CPU:2+"}
+    args = {
+        "name": "rh-cpu-multinode-777",
```

Comment:
Is 777 lucky?

Reply:
It was! But it's working now, so seems like we don't need luck any more ;)